vms/platformvm/block/builder/builder.go

```diff
@@ -154,7 +154,7 @@ func (b *builder) AddUnverifiedTx(tx *txs.Tx) error {
 			return err
 		}
 	}
-	return b.GossipTx(tx)
+	return b.GossipTx(context.TODO(), tx)
```

Comment:
nit: we usually use context.Background

Reply:
Will leave as TODO for now, it'll be bubbled up to `vm.go` as part of https://github.com/ava-labs/avalanchego/pull/2296 at which point we can properly fix